src/grid/_grid.scss

```diff
@@ -87,10 +87,7 @@
 
   // Define partial sizes for columnNumber < totalColumns.
   @for $i from 1 through ($grid-phone-columns - 1) {
-    .mdl-cell--#{$i}-col {
-      @include partial-size($i, $grid-phone-columns, $grid-phone-gutter);
-    }
-
+    .mdl-cell--#{$i}-col,
     .mdl-cell--#{$i}-col-phone.mdl-cell--#{$i}-col-phone {
```

Comment:
Leave the multiple declarations. It is to help address specificity issues.
On Jul 28, 2015 7:55 AM, "Neil" <notifications@github.com> wrote:

> In src/grid/_grid.scss
> <https://github.com/google/material-design-lite/pull/1213#discussion_r35638653>
> :
>
> > @@ -87,10 +87,7 @@
> >
> >    // Define partial sizes for columnNumber < totalColumns.
> >    @for $i from 1 through ($grid-phone-columns - 1) {
> > -    .mdl-cell--#{$i}-col {
> > -      @include partial-size($i, $grid-phone-columns, $grid-phone-gutter);
> > -    }
> > -
> > +    .mdl-cell--#{$i}-col,
> >      .mdl-cell--#{$i}-col-phone.mdl-cell--#{$i}-col-phone {
>
> Just noticed multiple selector declaration. Any reason behind this? Or I
> can change this into .mdl-cell-#{i}-col-phone only.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/material-design-lite/pull/1213/files#r35638653>
> .
>